hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/InternalOperationService.java

```diff
@@ -35,6 +35,11 @@
  */
 public interface InternalOperationService extends OperationService, PacketHandler {
 
+    /**
+     * Returns the size of the response queue.
+     *
+     * @return the size of the response queue.
```

Comment:
Isn't this redundant? I'd only use `@return` in more complicated cases, when there is something further to clarify in addition to the main description.